src/sentry/tasks/reports/deliver_organization_user_report.py

```diff
@@ -0,0 +1,132 @@
+from sentry import features
+from sentry.models import Organization, OrganizationStatus, Project, Team, User, UserOption
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.reports.types import Skipped
+from sentry.tasks.reports.utils.util import _to_interval, has_valid_aggregates
+from sentry.utils.compat import filter, zip
+
+DISABLED_ORGANIZATIONS_USER_OPTION_KEY = "reports:disabled-organizations"
+
+
+def user_subscribed_to_organization_reports(user, organization):
+    return organization.id not in (
+        UserOption.objects.get_value(user, key=DISABLED_ORGANIZATIONS_USER_OPTION_KEY)
+        or []  # A small number of users have incorrect data stored
+    )
+
+
+@instrumented_task(
+    name="sentry.tasks.reports.deliver_organization_user_report",
+    queue="reports.deliver",
+    max_retries=5,
+    acks_late=True,
+)
+def deliver_organization_user_report(
+    timestamp,
+    duration,
+    organization_id: int,
+    user_id: int,
+    dry_run: bool = False,
+) -> None:
+    from sentry.tasks.reports import backend, logger
+
+    try:
+        organization = Organization.objects.get(
+            status=OrganizationStatus.VISIBLE, id=organization_id
+        )
```

Comment:
Leaving a note here to check if this is equivalent, idk why it wasn't just copy/pasted from the original file. I'm nervous to make _any_ changes here since this was recently broken. 

This is what's in prod:
`organization = _get_organization_queryset().get(id=organization_id)` 

where `_get_organization_queryset` is:

```
def _get_organization_queryset():
    return Organization.objects.filter(status=OrganizationStatus.VISIBLE)
```

Reply:
For others: I put back `_get_organization_queryset` _just_ in case. I realize the end result of both queries is the same but given it took such a long time to figure out the oom error we were having with this task so recently, it doesn't seem worth it to me to make _any_ changes here.